kv/main.go

```diff
@@ -129,7 +131,7 @@ type generator struct {
 func newGenerator(seq *sequence) *generator {
 	return &generator{
 		seq:    seq,
-		rand:   rand.New(rand.NewSource(int64(time.Now().UnixNano()))),
+		rand:   rand.New(rand.NewSource(rand.Int63())),
```

Comment:
Do we seed the global rand? If we don't, then this is not doing what you expect.